config-model/src/test/java/com/yahoo/vespa/model/builder/xml/dom/ContentBuilderTest.java

```diff
@@ -206,7 +202,7 @@ public void handleSingleNonSearchPersistentVds() throws Exception {
         assertFalse(s.hasIndexedCluster());
         assertTrue(s.getClusters().isEmpty());
 
-        assertTrue(a.getPersistence() instanceof VDSEngine.Factory);
+        assertTrue(a.getPersistence() instanceof ProtonEngine.Factory);
```

Comment:
Test name is `handleSingleNonSearchPersistentVds` (emphasis "Vds"); should we remove it entirely or just rename it?